src/components/CustomModal/style.css

```diff
@@ -1,26 +1,29 @@
 /* Default and primary button colors differ within modals */
 .modal :global(.ant-modal-content) {
     /* prevents modal body color rendering under header/footer at corners */
-    border-radius: 3px;
     color: var(--light-theme-modal-color);
 }
 
 .modal :global(.ant-modal-header) {
     display: flex;
-    height: 46px;
     justify-content: center;
+    padding: 26px 24px 14px;
```

Comment:
Not sure if this is from the designs, but it might be nice to adjust the top margin so the `X` close button is aligned with the text

Reply:
(or change the margin of the close button below)

Reply:
The Overpass font rides a little bit high above the line, and the designs show the X icon set slightly higher than the text, at least to my eye.. hence the padding-bottom property on the close icon. I'm not opposed to making changes but that's what I was factoring in.